modules/devguide/examples/CasCheckAndSet.java

```diff
@@ -14,21 +14,17 @@
  * limitations under the License.
  */
 
-package com.couchbase.devguide;
+import java.util.concurrent.CountDownLatch;
 
 import com.couchbase.client.java.json.JsonArray;
-import com.couchbase.client.java.json.JsonObject;
 import com.couchbase.client.java.kv.GetResult;
 import com.couchbase.client.java.kv.ReplaceOptions;
 
-import java.util.concurrent.CountDownLatch;
-
-
 /**
- * Example of Cas (Check and Set) handling in Java for the Couchbase Developer Guide.
- * TODO: not tested
+ * Example of Cas (Check and Set) handling in Java for the Couchbase Developer
+ * Guide. TODO: not tested
```

Comment:
is the "TODO: not tested" comment still valid?

Reply:
Need to remove this, good shout!